marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/4/dhf.sjs

```diff
@@ -35,7 +36,7 @@ function run(context, func)
   if (label) {
     tracelib.setPluginLabel(label);
   } else {
-    fn.error(null, "DATAHUB-CONTEXT-MISSING-LABEL", "Your context object is missing a label");
+    http.throwBadRequest("Your context object is missing a label");
```

Comment:
Should this be httpUtils.throwBadRequest instead ?

Reply:
Thanks, I had some copy/paste errors there. Just fixed.